hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/ReconTaskSchemaDefinition.java

```diff
@@ -48,7 +50,9 @@
   @Override
   public void initializeSchema() throws SQLException {
     Connection conn = dataSource.getConnection();
-    createReconTaskStatus(conn);
+    if (!TABLE_EXISTS_CHECK.test(conn, RECON_TASK_STATUS_TABLE_NAME)) {
+      createReconTaskStatus(conn);
```

Comment:
Can we rename to createReconTaskStatusTable for consistency?

Reply:
Done.